GitUI/UserControls/FileStatusList.cs

```diff
@@ -56,6 +56,11 @@ public sealed partial class FileStatusList : GitModuleControl
         public new event EventHandler DoubleClick;
         public new event KeyEventHandler KeyDown;
         public new event EnterEventHandler Enter;
+
+        [Description("Disable showing open submodule menu items as bold")]
+        [DefaultValue(false)]
+        public bool DisableSubmoduleMenuItemBold { get; set; }
```

Comment:
A "bold" menu item implies "default" item that gets invoked on double click.
This property controls the visual aspect, i.e. item is bold or not, but I can't see where double-click action is enabled/disabled respectively.

Reply:
Implementation is in each form using FileStatusList, not related to the menu item (that item is unknown in the forms).